sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -961,7 +965,9 @@ public ReadAll<ParameterT, OutputT> withCoder(Coder<OutputT> coder) {
      * It should ONLY be used if the default value throws memory errors.
      */
     public ReadAll<ParameterT, OutputT> withFetchSize(int fetchSize) {
-      checkArgument(fetchSize > 0, "fetch size must be >0");
+      checkArgument(
```

Comment:
```suggestion
      // Note that api.java.sql.Statement#setFetchSize says it only accepts values greater than >= 0 and that MySQL supports using Integer.MIN_VALUE as a hint to stream the ResultSet instead of loading it into memory. See https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-implementation-notes.html for additional details.
      checkArgument(
```